### PR TITLE
feat: scaling ui

### LINKED ISSUE #2098: Enable zoom in/out the app
On windows CTRL + (Zoom in), CTRL - (Zoom out)
On MacOS CMD + (Zoom in), CMD - (Zoom out)

Expected behaviour as Discord:

https://user-images.githubusercontent.com/52249767/112141554-b374c000-8bd5-11eb-86f9-5e9159193119.mov

### PR DESCRIPTION
**Pull request by borismelnik**
fixes #2098

### CODE REVIEW

File: ui/app/AppMain.qml [hunk 1]
```diff
@@ -368,10 +368,11 @@ RowLayout {
...
     Rectangle {
         id: leftTab
-        Layout.maximumWidth: 78
-        Layout.minimumWidth: 78
-        Layout.preferredWidth: 78
+
+        property real tabButtonWidth: 20 * scaleAction.factor
+
         Layout.fillHeight: true
+        implicitWidth: tabButtonWidth + 36//78 * scaleAction.factor
```
Comment (jrainville) [inline ui/app/AppMain.qml:375]:
```suggestion
        implicitWidth: tabButtonWidth + 36
```

File: ui/nim-status-client.app/Contents/Info.plist [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jrainville) [inline ui/nim-status-client.app/Contents/Info.plist:1]: What does this do?

File: ui/qmake_qmake_immediate.qrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE RCC><RCC version="1.0">
```
Comment (jrainville) [inline ui/qmake_qmake_immediate.qrc:1]: Are those files needed for the scaling?

**Review by jrainville — COMMENTED** [review]
WOW
That's a lot of `scaleAction.factor`

Massive work.

I just left some small questions on the new auto-generated files to know if they are needed.

File: ui/app/AppMain.qml (hunk 1 above)
Comment (borismelnik) [reply ui/app/AppMain.qml:375]: its autogenerated file??

File: ui/nim-status-client.app/Contents/Info.plist (hunk 2 above)
Comment (borismelnik) [reply ui/nim-status-client.app/Contents/Info.plist:1]: sorry, i will remove it.

File: ui/qmake_qmake_immediate.qrc (hunk 3 above)
Comment (borismelnik) [reply ui/qmake_qmake_immediate.qrc:1]: i think i must fix gitignore. That is trash too/

**Comment by anastasiyaig** [comment]
missing scaling screens (no effect when `cmd  +`  or `cmd -` ):

1. select a random user when creating new account

Снимок экрана 2021-04-27 в 13 27 33


2. set a password screen

Снимок экрана 2021-04-27 в 13 25 23


3.

**Comment by iurimatias** [comment]
closing due to new incoming components in statusQ that will require redoing all this

**Pull request closed without merging**